src/org/thoughtcrime/securesms/loki/views/NewConversationButtonSetView.kt

```diff
@@ -209,23 +209,26 @@ class NewConversationButtonSetView : RelativeLayout {
     // region Interaction
     override fun onTouchEvent(event: MotionEvent): Boolean {
         val touch = PointF(event.x, event.y)
-        val expandedButton = expandedButton
         val allButtons = listOf( mainButton, sessionButton, closedGroupButton, openGroupButton )
         val buttonsExcludingMainButton = listOf( sessionButton, closedGroupButton, openGroupButton )
         if (allButtons.none { it.contains(touch) }) { return false }
         when (event.action) {
             MotionEvent.ACTION_DOWN -> {
+                if (isExpanded) {
+                    if (sessionButton.contains(touch)) { delegate?.createNewPrivateChat(); collapse()}
```

Comment:
Normally you don't count something as a click on `ACTION_DOWN`, but instead on `ACTION_UP`. In this case it works pretty nicely in practice though so it's fine :)